lib/raft/raft_impl.h

```diff
@@ -338,8 +338,8 @@ template <typename Server> class RaftImpl : public Raft<Server>
     if (!i_am_leader())
       return;
     // Attempt to pass leadership to a worthy successor.
-    const ::std::string *best_node = nullptr;
-    NodeState *best                = nullptr;
+    const ::std::string *best_node = NULL;
+    NodeState *best                = NULL;
```

Comment:
NBD, but this instance isn't from an upstream backport, right? This has been nullptr since inception in 38e1d741f2056a7893a0a90452d37a29ae9bf707, which goes back to 6.0.x?